pkg/volume/util/volumepathhandler/volume_path_handler_linux.go

```diff
@@ -47,6 +50,27 @@ func (v VolumePathHandler) AttachFileDevice(path string) (string, error) {
 	return blockDevicePath, nil
 }
 
+// DetachFileDevice takes a path to the attached block device and
+// detach it from block device.
+func (v VolumePathHandler) DetachFileDevice(path string) error {
+	loopPath, err := v.GetLoopDevice(path)
+	if err != nil {
+		if err.Error() == ErrDeviceNotFound {
+			klog.Warningf("DetachFileDevice: Couldn't find loopback device which takes file descriptor lock. device path: %q", path)
```

Comment:
logic in this func could be cleaned up a bit. For example I think this logs a warning and goes to the end of the function retuning `nil`. Is it supposed to just do nothing? It's unclear from the warning that this is ok (maybe it should say - "assuming file device already cleaned up" or osmething)